CODING_GUIDELINES.md

```diff
@@ -0,0 +1,79 @@
+# Background
+
+The goal of this guide is to capture some Do and Don'ts of Go code for the InfluxDB database. When it comes to Go, writing good code is often achieved with the help of tools like `go fmt` and `go vet`. However there are still some practices not enforceable any any tools. This guide lists some specific practices to follow when writing code for the database.
```

Comment:
"practices not enforceable **any any** tools" => practices not enforceable **by any** tools